freqtrade/persistence.py

```diff
@@ -156,6 +179,7 @@ class Trade(_DECL_BASE):
     __tablename__ = 'trades'
 
     id = Column(Integer, primary_key=True)
+    bot_id = Column(Integer, default=0, index=True)
```

Comment:
Any particular reason you choose Integer instead of string?
honestly, i would like to give my bots "names" ... not numbers ... but maybe i care too much for them ...

Reply:
good idea... will change it to string on the next commit

Reply:
yep "names" would be better :)